tests/IBI.WZDx.UnitTests/WzdxSerializerTests.cs

```diff
@@ -804,7 +806,15 @@ string expectedFeedJson
                                     Restrictions: new Restriction[]
                                     {
                                         new Restriction(RestrictionType.NoTrucks)
-                                    }
+                                    },
+                                    ImpactedCdsCurbZones: new CdsCurbZonesReference[]
+                                    {
+                                        new CdsCurbZonesReference(
+                                            CdsCurbZoneIds: new string[] {"Zone Id 1", "Zone Id 2"},

```

Comment:
For code style/consistency, there should be a space after/before the curly braces.